explorer/app/components/Section/Section.stories.tsx

```diff
@@ -1,25 +1,29 @@
+// Core dependencies
+import { Typography } from "@mui/material";
 import React from "react";
 import { ComponentStory, ComponentMeta } from "@storybook/react";
-import { Section } from "./Section";
+
+// App dependencies
+import { Section } from "./section";
```

Comment:
Is there an error with capitalization here? I'm getting the following error when I do `npm run storybook`:

```
ERROR in ./app/components/Section/Section.stories.tsx
Module not found: Error: [CaseSensitivePathsPlugin] `/Users/mim/git/cc-data-browser/explorer/app/components/Section/section.tsx` does not match the corresponding path on disk `Section.tsx`.
 @ ./app/components/Section/Section.stories.tsx 33:0-36 47:13-20 52:27-34
 @ ./app sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(js|jsx|ts|tsx))$
 @ ./generated-stories-entry.js
```